addons/hr_attendance/static/tests/hr_attendance_mock_server.js

```diff
@@ -1,32 +0,0 @@
-import { patch } from "@web/core/utils/patch";
-import { MockServer } from "@web/../tests/helpers/mock_server";
-
-patch(MockServer.prototype, {
-    /**
-     * Simulate the initialization of the attendance systray data
-     * @override
-     */
-    async _performRPC(route, args) {
-        if (route === "/hr_attendance/attendance_user_data") {
```

Comment:
Note: this was dead code, there is no test of the actual feature. If we want to test it some day, the new mock server should be adapted instead.